testonly/hammer/hammer.go

```diff
@@ -254,8 +254,9 @@ func HitMap(ctx context.Context, cfg MapConfig) error {
 
 // hammerState tracks the operations that have been performed during a test run.
 type hammerState struct {
-	cfg          *MapConfig
-	validReadOps *validReadOps
+	cfg            *MapConfig
+	validReadOps   *validReadOps
```

Comment:
Oh I see the `SetLeaves` thing below, so that's probably yes?

Reply:
It is indeed a yes. It could be that further refactoring will actually separate this such that write and read ops are never near each other and thus the "read" token in here seems spurious, but that's not the case at the moment so will leave for now.